share/spack/gitlab/cloud_pipelines/stacks/developer-tools/spack.yaml

```
@@ -4,7 +4,7 @@ spack:
     all:
       require: target=x86_64_v3
   concretizer:
-    reuse: true
+    reuse: fresh
```

```suggestion
    reuse: false
```

The issue in CI is coming from this. Looks like `reuse` is only a boolean option with `false` == `--fresh` on the command line.